hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -526,57 +500,98 @@ spec = around (showLogsOnFailure "EndToEndSpec") $ do
             BS.length logfile `shouldSatisfy` (> 0)
 
     describe "forking eras" $ do
-      it "does report on unsupported era" $ \tracer -> do
-        withClusterTempDir "unsupported-era" $ \tmpDir -> do
```

Comment:
Could: keep this test, but make it pending to re-enable when we can fork into the next era (after conway)